pkg/controllers/member/serviceexport/controller_integration_test.go

```diff
@@ -529,6 +533,27 @@ var _ = Describe("serviceexport controller", func() {
 			Eventually(serviceIsExportedFromMemberActual, eventuallyTimeout, eventuallyInterval).Should(Succeed())
 			Eventually(serviceIsExportedToHubActual(svc.Spec.Type, false, ptr.To(int64(newWeight))), eventuallyTimeout, eventuallyInterval).Should(Succeed())
 		})
+
+		It("invalid annotation weight should invalidate the exported service", func() {
```

Comment:
add a test, update the valid annotation to invalid

Reply:
nvm, i missed the create func in the beforeEach.

yeah, it's better to add add a case with "create export with invalid annotation" to make sure the serviceIsNotExportedActual